geowebcache/azureblob/pom.xml

```diff
@@ -23,6 +23,17 @@
       <artifactId>azure-storage-blob</artifactId>
       <!-- at the time of writing 11.0.1 was available but pom on repoes was corrupted -->
       <version>11.0.0</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+
+    <dependency>
```

Comment:
Hum hum... but this in turn depends on the slf4j-api no?
And the dependency management in the root pom should make sure they use the same version.

Reply:
I was not aware of ``<dependencyManagement>`` ability to override transitive version numbers.
I will revisit these exclusions with a transitive dependency in place.